webapp/src/components/properties/multiSelect.tsx

```diff
@@ -44,6 +42,9 @@ const MultiSelectProperty = (props: Props): JSX.Element => {
                         {v.value}
                     </Label>
                 ))}
+                {values.length === 0 && (
+                    <Label color='empty'>{''}</Label>
+                )}
```

Comment:
This is added so that if the selected row contains no values, the padding can still be retrieved. Otherwise, there is no way of determining the padding.